Android.bp

```diff
@@ -44,7 +44,10 @@ android_app {
     required: [
         "privapp_whitelist_com.stevesoltys.backup",
         "com.stevesoltys.backup_whitelist"
-    ]
+    ],
+    optimize: {
+        enabled: false,
```

Comment:
`java.lang.NoClassDefFoundError` was the issue right?

Could setup a custom `proguard.flags` in the future if needed.

Reply:
That actually had another reason, so we *could* re-enable proguard, but it is risky.

Reply:
Ah. I don't mind just leaving it disabled, definitely the better option for now.